migrations/V172__Add_Contract_Numbers_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS system_intake_contract_numbers (
+    id UUID PRIMARY KEY NOT NULL,
+    intake_id UUID REFERENCES system_intakes(id) ON DELETE CASCADE,
+    contract_number ZERO_STRING NOT NULL,
+    created_by TEXT NOT NULL CHECK (created_by ~ '^[A-Z0-9]{4}$'),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by TEXT CHECK (modified_by ~ '^[A-Z0-9]{4}$'),
+    modified_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
@ClayBenson94 , it would mess up base struct to leave them out. I've deviated away a few times and it generally has caused issue down the line. I suspect it's better to leave the modified meta data in place to support future auditing, and also to be able to note if a migration changed the data.

Reply:
I'm good with that!

@samoddball I'd say having this struct use BaseStruct (or the User Account base struct Steven mentioned) is best!

Reply:
@ClayBenson94 thank you! I worked with @StevenWadeOddball on Friday and got his new `BaseStructUser` integrated into here 😄